### PR TITLE
Beckys updates

### PR DESCRIPTION
**Pull request by beckybeauchamp1**

### CODE REVIEW

File: schedule.md [hunk 1]
```diff
@@ -1,52 +1,44 @@
...
 
-**Attendance will be taken at Outcomes (10:00pm) AND any scheduled one-on-ones.**
-
 | Time | Event |
 |------|-------|
-| 10:00 – 12:30pm | Outcomes Studio Tours |
-| 3:00 – 4:00pm | Mini-lesson: TBD |
-|*4:00 – 5:00pm*|*[One-on-ones](./one-on-ones.md)*|
+| 10:00 – 12:30pm | Outcomes |
+| 3:00 – 4:00pm | Mini Lesson: Many to Many Relationships |
```
Comment (nolds9) [inline schedule.md:27]: :+1: Outcomes confirmed Studio Tours

File: asking_for_help.md [hunk 2]
```diff
@@ -1,8 +1,13 @@
 # How to Ask Questions
 
+## Genius Bar
+The instructors will be offering support each day during `Genius Bar`. The schedule is outlined here: [schedule](./schedule.md) and on the WDI8 Calendar: [calendar](http://ga-dc.github.io/wdi8/calendar.html)
+
+Students will be limited to `5 minutes` at a time with one-on-one instructor support during `Genius Bar`. We are doing this to ensure we can support ALL students during a session as well giving you an opportunity to try working through your problem. You may have additional time with instructors during a session after an attempt to solve your problem :)
```
Comment (nolds9) [inline asking_for_help.md:6]: :+1:  Like it, just got to make sure instructors are on the same page, we use the timers!

**Comment by nolds9** [comment]
:+1:  looks great Becky!

File: asking_for_help.md (hunk 2 above)
Comment (jshawl) [inline asking_for_help.md:6]: maybe mention a waitlist too because students like that

**Comment by jshawl** [comment]
:+1:

**Pull request merged**